src/open_inwoner/conf/base.py

```diff
@@ -928,6 +929,95 @@
     }
 }
 
+#
+# django-setup-configuration
+#
+SETUP_CONFIGURATION_STEPS = [
+    "open_inwoner.configurations.bootstrap.zgw.ZakenAPIConfigurationStep",
+    "open_inwoner.configurations.bootstrap.zgw.CatalogiAPIConfigurationStep",
+    "open_inwoner.configurations.bootstrap.zgw.DocumentenAPIConfigurationStep",
+    "open_inwoner.configurations.bootstrap.zgw.FormulierenAPIConfigurationStep",
+    "open_inwoner.configurations.bootstrap.zgw.ZGWAPIsConfigurationStep",
+    "open_inwoner.configurations.bootstrap.kic.KlantenAPIConfigurationStep",
+    "open_inwoner.configurations.bootstrap.kic.ContactmomentenAPIConfigurationStep",
+    "open_inwoner.configurations.bootstrap.kic.KICAPIsConfigurationStep",
+]
+
+# ZGW configuration variables
+ZGW_API_CONFIG_ENABLE = config("ZGW_API_CONFIG_ENABLE", default=True)
+OIP_ORGANIZATION = config("OIP_ORGANIZATION", "")
+ZAKEN_API_ROOT = config("ZAKEN_API_ROOT", "")
+if ZAKEN_API_ROOT and not ZAKEN_API_ROOT.endswith("/"):
+    ZAKEN_API_ROOT = f"{ZAKEN_API_ROOT.strip()}/"
+ZAKEN_OAS_URL = (
+    ZAKEN_API_ROOT  # this is still required by the form, but not actually used
+)
+ZAKEN_API_CLIENT_ID = config("ZAKEN_API_CLIENT_ID", "")
+ZAKEN_API_SECRET = config("ZAKEN_API_SECRET", "")
+CATALOGI_API_ROOT = config("CATALOGI_API_ROOT", "")
+if CATALOGI_API_ROOT and not CATALOGI_API_ROOT.endswith("/"):
+    CATALOGI_API_ROOT = f"{CATALOGI_API_ROOT.strip()}/"
+CATALOGI_OAS_URL = (
+    CATALOGI_API_ROOT  # this is still required by the form, but not actually used
+)
+CATALOGI_API_CLIENT_ID = config("CATALOGI_API_CLIENT_ID", "")
+CATALOGI_API_SECRET = config("CATALOGI_API_SECRET", "")
+DOCUMENTEN_API_ROOT = config("DOCUMENTEN_API_ROOT", "")
+if DOCUMENTEN_API_ROOT and not DOCUMENTEN_API_ROOT.endswith("/"):
+    DOCUMENTEN_API_ROOT = f"{DOCUMENTEN_API_ROOT.strip()}/"
+DOCUMENTEN_OAS_URL = (
+    DOCUMENTEN_API_ROOT  # this is still required by the form, but not actually used
+)
+DOCUMENTEN_API_CLIENT_ID = config("DOCUMENTEN_API_CLIENT_ID", "")
+DOCUMENTEN_API_SECRET = config("DOCUMENTEN_API_SECRET", "")
+FORMULIEREN_API_ROOT = config("FORMULIEREN_API_ROOT", "")
+if FORMULIEREN_API_ROOT and not FORMULIEREN_API_ROOT.endswith("/"):
+    FORMULIEREN_API_ROOT = f"{FORMULIEREN_API_ROOT.strip()}/"
+FORMULIEREN_OAS_URL = (
+    FORMULIEREN_API_ROOT  # this is still required by the form, but not actually used
+)
+FORMULIEREN_API_CLIENT_ID = config("FORMULIEREN_API_CLIENT_ID", "")
+FORMULIEREN_API_SECRET = config("FORMULIEREN_API_SECRET", "")
+# ZGW config options
+ZAAK_MAX_CONFIDENTIALITY = config("ZAAK_MAX_CONFIDENTIALITY", None)
```

Comment:
Here you define the default for `ZAAK_MAX_CONFIDENTIALITY` as `None`. Later (in `bootstrap/zgw.py`) you set `config.zaak_max_confidentiality = settings.ZAAK_MAX_CONFIDENTIALITY`, but only if the latter is not `None`.

However, the default for  `config.zaak_max_confidentiality` (defined in `openzaak/models.py`) is `VertrouwelijkheidsAanduidingen.openbaar`. Why not use that default value here? Perhaps I don't fully understand the intended use of `django-setup-configuration`. Are we configuring these services in two steps?

Reply:
I did it like this, because otherwise I'd have to duplicate all the defaults set in `models.py` in this settings file. Although the likelihood of the defaults changing isn't that high, I'd rather just keep the defaults in one place (`models.py`)